packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Icon, Tooltip } from "@prismicio/editor-ui";
+import React, { useEffect, useState } from "react";
+import { Checkbox, Flex, Label } from "theme-ui";
+
+import { Col } from "@/legacy/components/Flex";
+
+type DisplayTextModel = { type: string };
+
+interface DisplayTextCheckboxProps {
+  text?: DisplayTextModel;
+  height?: 130 | 127;
```

Comment:
I wonder if we can avoid this, I don't really understand why we have to do it but it seems odd that we have these two values and then we use them as a percentage `height: `${height}%``

Reply:
@dani-mp I would like to avoid this, I don't like it. It's there to get the fields to align. The "130%" value aligns the checkbox for content relationships and link to media fields:

<img width="989" alt="Screenshot 2024-08-27 at 08 55 29" src="https://github.com/user-attachments/assets/bc086f5e-e3c0-48dd-8cf9-be5f64519680">

Then the "127%" option is needed for the link field. Without it, the fields are slightly misaligned: 

<img width="989" alt="Screenshot 2024-08-27 at 08 55 42" src="https://github.com/user-attachments/assets/193a4cda-9857-4d76-ba3a-15bbe31252b0">

I think it's just enough of a misalignment that it needs to be adjusted for. I can keep trying to find another solution for alignment. What do you think?

Reply:
not sure why this is happening, i guess it's a problem with the original layout and the intrinsic heights of those elements

i'm ok if we want to leave it like this as this is legacy code